.travis.yml

```diff
@@ -3,22 +3,27 @@ jdk:
   - oraclejdk8
 
 # Enable container-based architecture.
-sudo: false
+sudo: true
 
 addons:
   apt:
+    sources:
+      - ubuntu-toolchain-r-test
     packages:
       # Travis is on 64bit and there will be a cryptic aapt error w/o these libs.
       # For native code tests, we need some additional libraries if we are in a 64-bit environment.
       - libgd2-xpm
       - libc6:i386
       - libstdc++6:i386
       - zlib1g:i386
+      - g++-5
```

Comment:
This should have been done when Groovy was added too, but can you add a comment above this?  The native code tests comment above applies to the libraries above this, but not this one you are adding.